CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.1
+  - Bugs Fixed
+    -  Alchemy should not crash when there is a subgraph error (no network or The Graph servers are overloaded)
```

Comment:
Alchemy should not crash immediately when there is a subgraph error . it will retries for approximately 2.5 minutes.